rom/laghos_rom.hpp

```diff
@@ -602,6 +623,13 @@ class ROM_Basis
     void writeSP(ROM_Options const& input, const int window = 0) const;
     void readSP(ROM_Options const& input, const int window = 0);
 
+    void writePDweights(const int id, const int window = 0) const;
+
+    double GetOffsetX(const int idx)
```

Comment:
This could be a `const` function like the other `Get` functions.

Reply:
Made to `const`